mamba_ssm/modules/mamba2.py

```diff
@@ -31,10 +31,14 @@
 from mamba_ssm.ops.triton.ssd_combined import mamba_chunk_scan_combined
 from mamba_ssm.ops.triton.ssd_combined import mamba_split_conv1d_scan_combined
 
-from huggingface_hub import PyTorchModelHubMixin
 
-
-class Mamba2(nn.Module, PyTorchModelHubMixin):
+class Mamba2(
+        nn.Module,
+        PyTorchModelHubMixin,
```

Comment:
Per [this comment](https://github.com/state-spaces/mamba/pull/469#issuecomment-2230392664), this can be moved to the `MambaLMHeadModel` class